packaging/packager_win.bat

```diff
@@ -16,17 +16,22 @@ REM Requires VNC server and thus only available on Linux
 SET DISABLE_VNCDOTOOL=1
 
 REM Main deps
-cd %DISTRO_ROOT%\
+REM cd %DISTRO_ROOT%

```

Comment:
Just one quick comment, why is this commented out? It is not needed then?

Reply:
this is not needed

Reply:
I see, then let's remove it entirely.